### DISCUSSION TITLE
Request for Auto-Lock Feature for Magnetic Lock Integration

### DISCUSSION THREAD

**Discussion by mosshr** [rednblkx/HomeKey-ESP32#127] [Ideas]
I would like it to have an auto-lock function after a few seconds (maybe 3-5 seconds), to use with a magnetic lock.

**Comment by wxf18** [comment]
有什么办法，或者设置，让它解锁后自动锁定，谢谢

**Comment by rhamblen** [comment]
You can do that with an automation app like node red, or HA
On the MQTT message to open, start a timer and then trigger a lock command.

**Reply by wxf18** [reply]
后台自带的有，这个解决了，现在的新问题是待机功耗的问题，正常待机功耗130ma，怎么设置成超低功耗，用电池来供电，哪位大佬帮忙解决一下，谢谢！
